add-lure-modal.js

```diff
@@ -144,52 +106,15 @@ const AddLureModal = (props) => {
         <Text style={[text_style.sm, text_style.bold, margin_styles.bottom_md, text_style.primaryColor, text_style.alignCenter]}>
             {loadTranslations("requestNewLure")}
         </Text>
-
-        {reactIfView(currentTutorial == 'store', <Tooltip
-          contentStyle={[{backgroundColor: primary_color, height: 60}]}
-          backgroundColor={'rgba(0,0,0,0)'}
-          isVisible={currentTutorial == 'store'}
-          content={<Text style={[text_style.fontColorWhite]}>{loadTranslations("tutLureStore")}</Text>}
-          placement="top"
-          onClose={async () => {setCurrentTutorial(await updateTutorialAndGetNext('store', NAV_REQUEST_LURE_FORM))}}
-        >
-          <TooltipChildrenContext.Consumer>
-            {({ tooltipDuplicate }) => (
-              reactIfView(!tooltipDuplicate,
-                <View style={[{height:Platform.OS === 'android' ? 50 : 0, width: width}]}></View>
-              )
-            )}
-          </TooltipChildrenContext.Consumer>
```

Comment:
shouldnt these be replaced with TutorialTooltip

Reply:
That's weird, the conditions-results version of this file doesn't have the changes from lure-results branch. I'll take care of that